Source/Core/GeometryPipeline.js

```diff
@@ -1006,8 +1006,6 @@ define([
      * This is used by {@link Primitive} to efficiently render a large amount of static data.
      * </p>
      *
-     * @private
```

Comment:
I would remove the reference from `GeometryPipeline.transformToWorldCoordinates`:
```
This is used as a prerequisite to batch together several instances with {@link GeometryPipeline.combineInstances}.
```